ilert/README.md

```diff
@@ -1,41 +1,47 @@
 ## Overview
 
-The [iLert][1] integration sends Datadog alerts to iLert and seamlessly take actions on these alerts within the iLert platform.
+The [ilert][1] integration sends Datadog alerts to ilert and seamlessly takes actions on these alerts within the ilert platform.
 
-Integrate with iLert to:
+Integrate with ilert to:
 
 - Trigger and resolve incidents from Datadog
 - Tackle incidents and set up escalation policies as they occur
 - Set up a daily reminder of who is on-call
```

Comment:
The overview should include the following: 
What does your product (ilert) do (in 1-2 sentences)?
What will customers get from this integration, and why is it valuable to them?
What types of data will your integration send in or pull out, and what's the value of that data?